packages/lib/src/components/GooglePay/GooglePay.tsx

```diff
@@ -45,6 +46,11 @@ class GooglePay extends UIElement<GooglePayProps> {
     }
 
     public submit = () => {
+        // Analytics
+        if (this.props.isInstantPayment) {
+            this.submitAnalytics({ type: ANALYTICS_SELECTED_STR, target: 'instant_payment_button' });
+        }
```

Comment:
Same question that I asked on the ApplePay file applies here

Reply:
See previous answer